tasks/downloadEgibExcelTask.py

```diff
@@ -46,7 +46,12 @@ def run(self):
                         QgsMessageLog.logMessage('isCanceled')
                         return False
                     QgsMessageLog.logMessage('pobieram ' + url)
-                    service_api.retreiveFile(url=url, destFolder=self.folder, obj=self)
+                    res, exp = service_api.retreiveFile(url=url, destFolder=self.folder, obj=self)
+                    if not res:
+                        self.exception = exp
+                        return False
```

Comment:
Chyba coś qgis nie załapał tego exception
![image](https://github.com/user-attachments/assets/3cce98d0-198c-4398-a4d9-6a75361c4ff4)

Reply:
a to chyba nie do tego - bo tam masz bdot_task_finished